core/Nonce.php

```diff
@@ -103,7 +103,7 @@ public static function verifyNonceWithErrorMessage($id, $cnonce, $allowedReferre
         if (Url::isSecureConnectionAssumedByPiwikButNotForcedYet()) {
             $additionalErrors =  '<br/><br/>' . Piwik::translate('Login_InvalidNonceSSLMisconfigured',
                 array(
-                  '<a target="_blank" rel="noreferrer noopener" href="https://matomo.org/faq/how-to/faq_91/">',
+                  '<a target="_blank" rel="noreferrer noopener" href="' . Url::addCampaignParametersToMatomoLink('https://matomo.org/faq/how-to/faq_91/') . '">',
```

Comment:
Just a suggestion what this could look like:
```suggestion
                  Url::getExternalMatomoLinkTag('https://matomo.org/faq/how-to/faq_91/'),
```

Reply:
This would be a nice code improvement, I've created a [new issue](https://github.com/matomo-org/matomo/issues/21411) for it. 
`addCampaignParametersToMatomoLink()` will ignore non-matomo domains, so we could use the same method for all external links as there are quite a external links which go to non-Matomo domains. So maybe the new method could be called `getExternalLinkTag()`.